package.json

```diff
@@ -24,8 +23,8 @@
     "@types/node": "^14.0",
     "babel-jest": "^26.0.1",
     "jest": "^26.0.1",
-    "moxios": "^0.4.0",
-    "prettier": "^2.0.5"
+    "prettier": "^2.0.5",
+    "typescript": "^4.5.4"
```

Comment:
I added `typescript` as a dev dependency so it's clear what version this project is using.